packaging/msix/generate-placeholder-icons.py

```
@@ -0,0 +1,72 @@
+except ImportError:
+    print("Pillow not installed. Install with: pip install Pillow")
+    print("Or create PNG files manually at these sizes in Assets/:")
+    print("  Square44x44Logo.scale-100.png    (44x44)")
+    print("  Square44x44Logo.scale-200.png    (88x88)")
+    print("  Square44x44Logo.targetsize-44_altform-unplated.png (44x44)")
+    print("  Square150x150Logo.scale-100.png  (150x150)")
+    print("  Square150x150Logo.scale-200.png  (300x300)")
+    print("  Wide310x150Logo.scale-100.png    (310x150)")
+    print("  Wide310x150Logo.scale-200.png    (620x300)")
+    print("  StoreLogo.scale-100.png          (50x50)")
+    raise SystemExit(1)
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)